code/modules/mob/living/carbon/human/species.dm

```diff
@@ -445,6 +460,10 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		C.Digitigrade_Leg_Swap(TRUE)
 	for(var/X in inherent_traits)
 		REMOVE_TRAIT(C, X, SPECIES_TRAIT)
+	for(var/obj/item/organ/external/organ in C.internal_organs)
```

Comment:
Why are external organs in internal organs?

Reply:
All organs are stored in internal_organs, even alien emryo's, eyes, ears, etc. They don't really need to be there, but I don't think it's necessary to make an exception (I should probably check if they show up in organ manipulation though...)

Reply:
I see. Just a case of weird naming.